charts/sonarqube/README.md

```diff
@@ -29,6 +33,19 @@ The above command deploys SonarQube on the Kubernetes cluster in the default con
 
 The default login is admin/admin.
 
+## Installing the SonarQube 9.9 LTS chart
+
+The version of the chart for the SonarQube 9.9 LTS is being distributed as the `8.x.x` version of this chart.
+
+In order to use it, please set the version constraint `~8`, which is equivalent to `>=8.0.0 && <= 9.0.0`. That version parameter **must** be used in every helm related command including `install`, `upgrade`, `template`, and `diff` (don't treat this as an exhaustive list).
+
+Example:
+```
+helm upgrade --install -n sonarqube --version ~8 sonarqube sonarqube/sonarqube
+```
+
+To upgrade from the old and unmaintained [sonarqube-lts chart](https://artifacthub.io/packages/helm/sonarqube/sonarqube-lts), please follow the steps described [in this section](#upgrade-from-sonarqube-lts-to-this-chart).
```

Comment:
The link #upgrade-from-sonarqube-lts-to-this-chart is broken. It might be the case for some others since we changed a few titles in various files.